### PR TITLE
Create 7Dtd Template

### PR DESCRIPTION
**Pull request by LexFuturorum**
7 Days to Die Template
Readme if following up

### CODE REVIEW

File: 7DtD.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            }
+        ],
+        "environmentVars": {}
+    },
+    "supportedEnvironments": [
+        {
+            "type": "tty"
+        }
+    ],
+    "name": "7DtD"
```
Comment (LordRalex) [inline 7DtD.json:88]: Your JSON is not valid down here.

**Review by LordRalex — CHANGES_REQUESTED** [review]

**Comment by LordRalex** [comment]
I thought 7D2D was a lot easier to use. I recall doing this in the past, and I did not need to do telnet or anything. What is up with that?

File: 7DtD.json (hunk 1 above)
Comment (LexFuturorum) [reply 7DtD.json:88]: i was wondering whats the problem with the code there for a sec.
thx ill fix that

**Comment by LordRalex** [comment]
So, seems like that's how you are meant to access it... that's terrible....
Does having the script just auto-run telnet and connecting there not work here? I used https://7daystodie.gamepedia.com/Server as a reference, and they run telnet within the same script, which seems reasonable to me too. 
However, I don't see this working well for shutting it down.

**Comment by LexFuturorum** [comment]
> I thought 7D2D was a lot easier to use. I recall doing this in the past, and I did not need to do telnet or anything. What is up with that?

telnet is a handy way of getting a live console

**Comment by LexFuturorum** [comment]
> So, seems like that's how you are meant to access it... that's terrible....
> Does having the script just auto-run telnet and connecting there not work here? I used https://7daystodie.gamepedia.com/Server as a reference, and they run telnet within the same script, which seems reasonable to me too.
> However, I don't see this working well for shutting it down.

Shutdown works better than the non reaction i got before however kill still doesnt work because it only kills the telnet client
also the script u send is windows and basicly does the same as my template i only offloaded it to pufferpanel

**Pull request closed without merging**